docs/user-guide/compute-resources.md

```diff
@@ -203,6 +203,10 @@ Containers:
       memory:       50Mi
     State:      Running
       Started:      Tue, 07 Jul 2015 12:54:41 -0700
+		Last Termination State:     Terminated
+			Exit Code:                137
```

Comment:
Is the spacing on this correct? Shouldn't `Last Terminations State` be aligned with `State`?